toolchain/check/testdata/basics/no_prelude/textual_ir.carbon

```diff
@@ -2,9 +2,9 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
-// ARGS: compile --no-prelude-import --phase=check --dump-sem-ir %s
+// EXTRA-ARGS: --dump-sem-ir
 //
-// Check that the command-line flag to dump textual IR works.
```

Comment:
This test seems more clear using the old form where it directly passed the flag and didn't get any "default" arguments that might obscure the behavior.

For example, this wouldn't fail if we renamed the flag and made the old spelling a no-op?

Reply:
I'm not sure what's best here. This is a check test not a driver test, so I was interpreting the intent as being to test that we get textual IR out rather than to test the flag itself. But it's testing the behavior that we enable by default for check's file_test tests -- and the SemIR dumping logic also isn't in check.

Hm, maybe these tests are really driver tests, and should be moved to `toolchain/driver` instead and switched back to using `ARGS`. I'll look into that.

Reply:
Done.